config/config.go

```diff
@@ -53,9 +53,9 @@ type BlockManagerConfig struct {
 	// DABlockTime informs about block time of underlying data availability layer
 	DABlockTime time.Duration `mapstructure:"da_block_time"`
 	// DAStartHeight allows skipping first DAStartHeight-1 blocks when querying for blocks.
-	DAStartHeight uint64            `mapstructure:"da_start_height"`
-	NamespaceID   types.NamespaceID `mapstructure:"namespace_id"`
-	FraudProofs   bool              `mapstructure:"fraud_proofs"`
+	DAStartHeight    uint64            `mapstructure:"da_start_height"`
+	NamespaceID      types.NamespaceID `mapstructure:"namespace_id"`
+	StateFraudProofs bool              `mapstructure:"fraud_proofs"`
```

Comment:
changing ``mapstructure:"fraud_proofs"`` to ``mapstructure:"state_fraud_proofs"`` would make sense as well.